### DISCUSSION TITLE
Different Colours for the Electribe Display

### DISCUSSION THREAD

**Discussion by maxtribe** [bangcorrupt/hacktribe#111] [General]
@bangcorrupt 

Hi

I just saw in another post that you can do different key combinations before starting and that you get into a menu with it. I was also able to see different colors. I wanted to ask the following. Is it possible to change the color of the display permanently? Because that would also be damn cool if it had a different color while you were programming.

And secondly, I would also be interested. What can I do with the key combinations before starting the Electribe, or better said, what do they do?

Thank you for the information.

Greetings Max

**Comment by bangcorrupt** [comment]
See [this comment](https://github.com/bangcorrupt/hacktribe/discussions/109#discussioncomment-2051265) and [Test Mode](https://github.com/bangcorrupt/hacktribe/wiki/Test-Mode) in the wiki.

**Comment by Maxklui** [comment]
@bangcorrupt I tried out different types of operations on the Electribe but every operation what I want to do Shows me different types of Error. Can you say me maybe how can I fix this.

Thank you in advanced
